com.unity.perception/Editor/Randomization/Editors/RunInUnitySimulationWindow.cs

```diff
@@ -166,6 +168,15 @@ static string IncrementRunName(string runName)
 
         async void RunInUnitySimulation()
         {
+            #if PLATFORM_CLOUD_RENDERING
```

Comment:
What is the intended use case here? Only internal devs will have access to CloudRendering build target, so this enhancement doesn't appear to be for users. Is that your intent with this change? I was under the impression you would also add support for selecting an existing build id so that we can use builds uploaded from Cloud Build.

Reply:
That is correct, for now, only internal devs and on-prem customers who have access to the cloud rendering build target will have an option to switch to cloud rendering and then do the build and run on usim. The cloudbuild workflow is still not finalized. There are a lot of friction points with that workflow and would require some work on the cloud build side as well. For now the focus is on on-prem customers for the advantaged SKU and internal usage.

Reply:
Ok. Can follow up offline